React/Modules/MacOS/RCTAccessibilityManager.m

```diff
@@ -54,6 +54,14 @@ - (void)dealloc
   callback(@[ @(isVoiceOverEnabled) ]);
 }
 
+RCT_EXPORT_METHOD(setAccessibilityFocus:(nonnull NSNumber *)reactTag)
+{
+   dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Should we use `RCTExecuteOnMainQueue` here?
```suggestion
   RCTExecuteOnMainQueue(^{
```
(You might have to also add `#import "RCTUtils.h"` on top.)

Reply:
It is consistent with the iOS implementation that just calls dispatch_async on main queue.   The RCTExecuteOnMainQueue does the same thing, but checks if already on the main queue and if so just calls the block.   In this case it will *always* be called on the JS thread.